### DISCUSSION TITLE
Models architecture and params

### DISCUSSION THREAD

**Discussion by Cirediallo** [k2-fsa/icefall#1087] [Q&A] [answered]
Hi,
where can we find models architecture and params for all differents receipes. 
Thank you.

**Comment by csukuangfj** [comment, marked as answer]
In RESULTS.md of each recipe, for instance
https://github.com/k2-fsa/icefall/blob/master/egs/librispeech/ASR/RESULTS.md

![Screenshot 2023-05-23 at 17 50 52](https://github.com/k2-fsa/icefall/assets/5284924/db6568ca-2250-489d-a244-1010529fcca4)
